db/schema.rb

```diff
@@ -10,12 +10,14 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_08_12_143158) do
+ActiveRecord::Schema.define(version: 2022_09_06_140208) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "btree_gist"
+  enable_extension "citext"
```

Comment:
A couple of random extensions snuck in - this and `uuid-ossp` below.

Reply:
Ok, should have snuck them out now =)